### DISCUSSION TITLE
环境变量配置失效了

### DISCUSSION THREAD

**Discussion by blcyzycc** [alex8088/electron-vite#378] [General]
配置如图所示，但是在页面打印 import.meta.env 中，没有设置的环境内容
![微信图片_20240110093530](https://github.com/alex8088/electron-vite/assets/36541261/a26da210-9d63-4b04-9379-d741633fafd1)

**Comment by alex8088** [comment]
https://cn.electron-vite.org/guide/env-and-mode

**Reply by blcyzycc** [reply]
仔细查看图片呢，我这个配置就是按照文档里面配置的。或者有没有带环境变量的开源仓库地址，我看看

**Reply by alex8088** [reply]
```
KEY=123                # 无效变量
MAIN_VITE_KEY=123      # 仅主进程可用
PRELOAD_VITE_KEY=123   # 仅预加载脚本可用
RENDERER_VITE_KEY=123  # 仅渲染器可用
VITE_KEY=123           # 所有进程共用
```
没有正确使用前缀，注意 `VITE_` 需 2.0.0 版本起才支持

**Reply by blcyzycc** [reply]
OK
